### DISCUSSION TITLE
Feature request: a built-in collection of atmospheric music that can be used as a background

### DISCUSSION THREAD

**Discussion by sherlock440** [TortugaPower/BookPlayer#570] [Ideas]
Some readers have a cool feature: they allow you to add subtle background atmospheric music (like the burning of a campfire or the pounding of rain).

It would be very cool if this app had a similar option, at least for those who have paid for it. Well, and users should be able to adjust its volume

**Comment by pichfl** [comment]
Crazy idea. I like it! Especially for old recordings. 

- Where would we source the necessary sounds? We can't infringe on copyright and our assets are always out in the open which makes it hard to license things.
- How do we provide them without bloating the basic app (most audio files would be larger than the total size of the compiled app as it is right now)

**Reply by sherlock440** [reply]
What if we just let users add a soundtrack to each particular audiobook, allowing them to adjust its volume?

To do this, you could make an additional button in the user's audiobooks directory. Well, if an audio track is added to a book, then display an additional button in the player for that particular book to adjust the volume

**Reply by pichfl** [reply]
Good solution 👍

**Reply by Yonidass** [reply]
You can already do this with iOS' built in Background Sounds. It's limited to a just a few white noise/nature backgrounds, and I don't think you can adjust the volume, but it works fine while listening to audiobooks.

**Comment by yuriythebest** [comment]
That's insane! I wanted to suggest just this exact idea and bam! here it is! Great minds think alike!
I really like to listen to 8bit music as background noise, so it would be great to be able to have some way to designate some imported audio files as background music in a special folder, and be able to adjust the volume separately.
